app/controllers/application_controller.rb

```diff
@@ -1,2 +1,10 @@
 class ApplicationController < ActionController::Base
+  before_action :configure_permitted_parameters, if: :devise_controller?
```

Comment:
- [ ] @Team Great work adding devise, however, you forgot to add `before_action :authenticate_user!` so that devise can authenticate users accessing the application